src/main/java/seedu/address/model/team/Task.java

```diff
@@ -100,7 +103,15 @@ public String getDeadlineAsString() {
         if (deadline == null) {
             return "";
         } else {
-            return String.format("(By %s)", deadline.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")));
+            return String.format(" (By %s)", deadline.format(DateTimeFormatter.ofPattern(DATE_FORMAT)));
```

Comment:
See if this works better
```suggestion
            return String.format(" (By %s)", deadline.format(DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm a")));
```